common/lib/xmodule/xmodule/templates/problem/checkboxes_response.yaml

```diff
@@ -3,21 +3,46 @@ metadata:
     display_name: Checkboxes
```

Comment:
@sarina thanks. yes, I'll punt my decision until I see a sandbox...much easier for me to make a decision with context.